apps/daimo-mobile/src/logic/inAppNotifications.ts

```diff
@@ -54,11 +54,12 @@ export function useInAppNotifications(): InAppNotificationInfo {
 
     // Requests
     for (const request of notificationRequestStatuses) {
-      // Skip requests we've declined or cancelled.
+      // Skip requests we've declined, cancelled or sent as link.
       if (
         request.status === DaimoRequestState.Cancelled ||
         (request.status === DaimoRequestState.Declined &&
-          request.recipient.addr !== account.address)
+          request.recipient.addr !== account.address) ||
+        !request.expectedFulfiller
```

Comment:
recommend showing outbound links eg

> "You requested $5.00 via link"

Reply:
done in latest commit
<img width="256" src="https://github.com/daimo-eth/daimo/assets/6984346/fba4a65f-6c47-4ed3-bd89-d3e3611083ae">